src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudOneShotInstanceIT.java

```diff
@@ -99,7 +103,7 @@ public static void init() throws Exception {
     Builder otherStep = execute(Commands.ECHO, "\"also works\"");
     otherProject.getBuildersList().add(otherStep);
     otherProject.setAssignedLabel(new LabelAtom(LABEL));
-    otherBuildFuture = otherProject.scheduleBuild2(0);
+    otherBuildFuture = otherProject.scheduleBuild2(30);
```

Comment:
Suggest using a constant here, QUIET_PERIOD_SECS, or else leaving an in-line comment explaining what this literal represents.